src/test/java/teammates/test/cases/util/TimeHelperTest.java

```diff
@@ -65,6 +65,16 @@ public void testEndOfYearDates() {
         assertEquals("30 Dec 12:00 NOON", TimeHelper.formatDateTimeForInstructorHomePage(date));
     }
 
+    @Test
+    public void testFormatDateForInstructorPages() {
+        ZoneId zoneId = ZoneId.of("UTC");
+        Instant instant = LocalDateTime.of(2018, Month.AUGUST, 18, 0, 0).atZone(zoneId).toInstant();
+        assertEquals("18 Aug 2018", TimeHelper.formatDateForInstructorPages(instant, zoneId));
+
+        instant = LocalDateTime.of(2018, Month.AUGUST, 18, 12, 10).atZone(zoneId).toInstant();
+        assertEquals("18 Aug 2018", TimeHelper.formatDateForInstructorPages(instant, zoneId));
```

Comment:
Honestly I don't think tests are necessary since these are facades 😅 the burden of testing is on the team that maintains `java.time` (and I'm sure it's well-tested)

Reply:
The tests can be written as _integration_ tests i.e., just to confirm that the facade is calling the right method of the java.time API. no need to test many combinations, which is what we do when _unit_ testing.

Reply:
I am of the opinion that we can just do with the first test case since that'll be sufficient to verify the right method is called.

Reply:
Tiebreaker from @xpdavid or @wkurniawan07?

Reply:
The second test might not be necessary.

It is calling `formatInstant()` which is a private method and we should check it indeed calls the private method. Since there are some uncovered parts for this private method. Why not change the second test (e.g. null input and should return empty string) to cover the part? 

https://codecov.io/gh/TEAMMATES/teammates/src/master/src/main/java/teammates/common/util/TimeHelper.java#L333

Reply:
Sure. So in this case is it better if we test the "null" input in `testFormatDateTimeForDisplay` rather than `testFormatDateForInstructorPages`?

Reply:
I mean we still need to the first case. But instead of removing the second test, let modify the second test case to test the null input.

Reply:
Sure @IAmPramod Let's do it then!

Reply:
@dalessr will do it then!

Reply:
@dalessr @xpdavid I have made the changes in the test case.